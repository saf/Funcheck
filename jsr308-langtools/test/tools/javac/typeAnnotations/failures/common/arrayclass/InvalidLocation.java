/*
 * @test /nodynamiccopyright/
 * @bug 6843077
 * @summary check for invalid annotatins given the target
 * @author Mahmood Ali
 * @compile/fail/ref=InvalidLocation.out -XDrawDiagnostics -source 1.7 InvalidLocation.java
 */

class InvalidLocation {
  void test() {
    Object a = String @A [].class;
  }
}

@java.lang.annotation.Target(java.lang.annotation.ElementType.TYPE)
@interface A { }
