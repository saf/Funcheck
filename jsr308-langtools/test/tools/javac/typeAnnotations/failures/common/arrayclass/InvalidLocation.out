InvalidLocation.java:11:23: compiler.err.annotation.type.not.applicable
1 error
